src/Faithlife.Data/DbValueTypeInfo.cs

```diff
@@ -104,7 +104,7 @@ public T GetValue(IDataRecord record, int index, int count)
 			else if (m_strategy == DbValueTypeStrategy.ByteArray)
 			{
 				if (record.IsDBNull(index))
-					return default;
+					return default!;
```

Comment:
Can `!` be avoided with `[return: MaybeNull]`?

Reply:
Not IIRC, but I'll double-check.

Reply:
Confirmed. Doesn't seem to help.

Reply:
Hmm, I'm surprised, but I'd have to check it out in a small repro.